x/group/types/config.go

```diff
@@ -0,0 +1,35 @@
+package types
+
+import "time"
+
+// Config is a config struct used for intialising the group module to avoid using globals.
```

Comment:
The first words are redundant and could be made concise by
`// Config is used to initialize and customize the group module.`